code/game/objects/items/weapons/design_disks/NeoTheology.dm

```diff
@@ -25,6 +25,13 @@
 		/datum/design/autolathe/device/grenade/nt_cleaner,
 		/datum/design/autolathe/device/grenade/nt_weedkiller,
 
+		/datum/design/bioprinter/nt_cells/large,
+		/datum/design/bioprinter/nt_cells/large/plasma,
+		/datum/design/bioprinter/nt_cells/medium,
+		/datum/design/bioprinter/nt_cells/medium/plasma,
+		/datum/design/bioprinter/nt_cells/small,
+		/datum/design/bioprinter/nt_cells/small/plasma,
```

Comment:
Might be a better idea to put them as EOTP reward instead.

Reply:
That will make them unavailable most of the time tho.